app/gui/src/controller/graph/widgets.rs

```diff
@@ -0,0 +1,481 @@
+//! Widgets controller.
```

Comment:
if this file does not contain multiple widgets, it should be named `widget`. We are not using plural names if the file does not contain plural definitions of a given thing.